src/job/stagingJobHandler.ts

```diff
@@ -72,6 +72,14 @@ export class StagingJobHandler extends JobHandler {
       if (resp?.output?.includes('Summary')) {
         resp.output = resp.output.slice(resp.output.indexOf('Summary'));
       }
+      // Invoke Gatsby Preview Webhook
+      const featurePreviewWebhookEnabled = process.env.GATSBY_CLOUD_PREVIEW_WEBHOOK_ENABLED;
+      if (featurePreviewWebhookEnabled) {
+        // TODO: We may want to send a response to slack or something to let us know
+        // the outcome of the call
+        this.previewWebhook();
```

Comment:
Agree with your TODO that we should probably be logging / slacking / something-ing here to capture the response from the post to the webhook. Do you want to just add some rudimentary logging for now?